docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -25,12 +25,19 @@ Existing validators propose and vote to
 [add or remove validators](Add-Validators.md#ibft-20). Adding or removing a validator
 requires a majority vote (greater than 50%) of validators.
 
-## Minimum number of validators
+## Validators
+
+### Minimum number of validators
 
 IBFT 2.0 requires four validators to be Byzantine fault tolerant. Byzantine fault tolerance is the
 ability for a blockchain network to function correctly and reach consensus despite nodes failing or
 propagating incorrect information to peers.
 
+### Maximum number of validators
+
+As the number of validators increase, the message complexity increases as well, which can cause a loss of performance.
+In [network tests](https://wiki.hyperledger.org/display/BESU/Maximum+Validator+count+for+an+IBFT2+Network), IBFT 2.0 handles up to 30 validators safely with no loss of performance.
+
```

Comment:
Is it worth clarifying here that the number of non-validator nodes doesn’t affect the performance?

Reply:
I'm concerned that someone might read this to mean you can only have max 30 nodes in a besu network